packages/runtime/test/transport/relationships.test.ts

```diff
@@ -350,6 +355,59 @@ describe("Relationship status validations on active relationship", () => {
     });
 });
 
+describe("tests on inactive relationships", () => {
```

Comment:
1. I don't like this describe name
2. shouldn't this be inside the message test file?